core/src/main/java/com/novoda/noplayer/mediaplayer/PlaybackStateChecker.java

```diff
@@ -0,0 +1,33 @@
+package com.novoda.noplayer.mediaplayer;
+
+import android.media.MediaPlayer;
+
+import static com.novoda.noplayer.mediaplayer.PlaybackStateChecker.PlaybackState.ERROR;
+import static com.novoda.noplayer.mediaplayer.PlaybackStateChecker.PlaybackState.IDLE;
+import static com.novoda.noplayer.mediaplayer.PlaybackStateChecker.PlaybackState.PREPARING;
+
+class PlaybackStateChecker {
+
+    boolean isPlaying(MediaPlayer mediaPlayer, PlaybackState playbackState) {
```

Comment:
None of these complex statements are in the `AndroidMediaPlayerFacade` anymore allowing us to simply mock and return either `true` or `false`.